standalone/src/main/scala/filodb.standalone/FiloServer.scala

```diff
@@ -90,10 +91,6 @@ class FiloServer(watcher: Option[ActorRef]) extends FilodbClusterNode {
 
 object FiloServer extends StrictLogging {
   def main(args: Array[String]): Unit = {
-    try {
-      new FiloServer().start()
-    } catch { case e: Exception =>
-      logger.error("Could not start FiloDB server", e)
-    }
+    new FiloServer().start()
```

Comment:
We need a force shutdown here. There is a possibility of daemon threads holding up the process.

Reply:
The force shutdown is already present at https://github.com/filodb/FiloDB/pull/1124/files#diff-c50749986da7e22c6df371808ceb7bfed314508b866b9a0db2d9bcd4727165e1R72-R75

Do we still need to add it here?

Reply:
Ah got it. Do you mind moving the try-catch here to the uppermost call - that way any future additions here will also benefit from it. Thanks.